### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by wsdjeg**

### CODE REVIEW

**Comment by Shougo** [comment]
It should be merged after #602.

**Comment by wsdjeg** [comment]
yeah, and I think maybe a LOGO will be better than

```
# deoplete
```

on the top of readme.

**Comment by Shougo** [comment]
LOGO PR is wellcome.

File: README.md [hunk 1]
```diff
@@ -1,16 +1,17 @@
...
 [![Join the chat at https://gitter.im/Shougo/deoplete.nvim](https://badges.gitter.im/Shougo/deoplete.nvim.svg)](https://gitter.im/Shougo/deoplete.nvim?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+[![Doc](https://img.shields.io/badge/doc-%3Ah%20deoplete-orange.svg)](doc/deoplete.txt)
 
 Deoplete is the abbreviation of "dark powered neo-completion".  It
-provides an asynchronous keyword completion system in the
-current buffer.
-To view the current options, please consult the
-[documentation](https://github.com/Shougo/deoplete.nvim/blob/master/doc%2Fdeoplete.txt).
+provides an extensible and asynchronous completion framwork for
+neovim/vim8.
```
Comment (Shougo) [inline README.md]: It should be `neovim/Vim8`

Comment (wsdjeg) [reply README.md]: Yeah, I will Update it  this PR is still WIP, and I am using github web editor.  and when done, I will squash all commits.

**Comment by Shougo** [comment]
@wsdjeg  Ping.
Please fix the conflict.

I will merge it later.

**Comment by wsdjeg** [comment]
Ok, I will finish this PR tonight.

**Comment by wsdjeg** [comment]
@Shougo The commits is too many, I will squash commits.

**Comment by Shougo** [comment]
OK.  Squashed.

**Pull request merged**